addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoBridgeHandler.java

```diff
@@ -85,6 +90,18 @@ private String getApiScope() {
             stringBuilder.append("read_thermostat write_thermostat ");
         }
 
+        if (configuration.readWelcome) {
+            stringBuilder.append("read_camera ");
+        }
+
+        if (configuration.accessWelcome) {
+            stringBuilder.append("access_camera ");
+        }
+
+        if (configuration.writeWelcome) {
+            stringBuilder.append("write_camera ");
+        }
+
```

Comment:
I thought so because what he does with it depends upon what's done with it's items, rules.... Thus the user give visibility/access on what he wants. But it's your decision.